resources/views/admin/error.blade.php

```diff
@@ -0,0 +1,215 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Title</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet"
+        href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+        crossorigin="anonymous">
+    <style>
+    @import url('https://fonts.googleapis.com/css?family=Montserrat:400,600,700');
+    @import url('https://fonts.googleapis.com/css?family=Catamaran:400,800');
+
+    .error-container {
```

Comment:
Should we create a new CSS file and put these styles into it? @ntu34543

Reply:
Done!